src/display/screen.js

```diff
@@ -11,10 +11,10 @@ var Screen = function(canvas, pixelSize) {
 };
 
 Screen.colors = [
-    0xFF,
-    0xAA,
-    0x55,
-    0x00
+    [0xFF, 0xFF, 0xFF, 0xFF],
+    [0xAA, 0xAA, 0xAA, 0xAA],
+    [0x55, 0x55, 0x55, 0x55],
+    [0x00, 0x00, 0x00, 0x00]
```

Comment:
I guess we should have only 3 values by vector for RGB ?
Also, can you add a small comment to explicitly indicate that this array is using RGB values ?

Reply:
Oops, I am too used to manipulate RGBA values ^^'